README.md

```diff
@@ -16,14 +16,15 @@ use man::prelude::*;
 
 fn main() {
   let page = Manual::new("basic")
-    .about("A basic example")
+    .about("A basic example".into())
```

Comment:
Would we not want to prefer taking an Into<String> as an argument for `about` instead of making users do this?

Reply:
It's probably worth it, but until then the main example in the README doesn't work.

Reply:
Merged the PR. Happy to either merge this PR as-is or wait for an update. Up to you haha.